src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -201,7 +201,7 @@ class BasePaymentsPage extends React.Component {
     }
 
     fetchData() {
```

Comment:
I might just be missing it, but I'm not seeing the use of the onyx key `IS_LOADING_PAYMENT_METHODS` here - are we using it?

Reply:
Its going to be used in KYCWall. Here and another spot.